user_guide.adoc

```diff
@@ -3363,8 +3364,8 @@ MyCustomProcessor _process__raw_buf = new MyCustomProcessor(key());
 this.buf = _process__raw_buf.decode(this._raw_buf);
 ----
 
-Typical implementation of a custom processor would look like that
-(again, example for in Java, refer to language-specific notes for
+A typical implementation of a custom processor would look like this
+(again, example in Java; refer to language-specific notes for
```

Comment:
A random thought (I do not suggest to address it in this PR at all): I remember that @GreyCat somewhere said it would be nice to show all supported target languages (i.e. C++, C#, Python, ...) for all code samples which show the generated code in one particular target language (mostly Java, currently), using tabs for switching among them.

Reply:
Sounds a good idea.